lib/completion.js

```diff
@@ -79,8 +79,10 @@ module.exports = function completion (yargs, usage, command) {
 
   // generate the completion script to add to your .bashrc.
   self.generateCompletionScript = function generateCompletionScript ($0, cmd) {
+    let zshShell = process.env.SHELL && process.env.SHELL.indexOf('zsh') !== -1
```

Comment:
I believe this could be `const` and not `let`.

Reply:
fixed.